twc/src/main/java/org/openmbee/sdvc/twc/twcrevisionmmscommitmap/TwcRevisionMmsCommitMapController.java

```diff
@@ -0,0 +1,29 @@
+package org.openmbee.sdvc.twc.twcrevisionmmscommitmap;
+
+import org.openmbee.sdvc.core.objects.CommitsResponse;
+import org.openmbee.sdvc.crud.controllers.BaseController;
+import org.openmbee.sdvc.twc.services.TwcRevisionMmsCommitMapService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/projects/{projectId}/commits/{commitId}")
+public class TwcRevisionMmsCommitMapController extends BaseController {
+
+    private TwcRevisionMmsCommitMapService twcRevisionMmsCommitMapService;
+
+    @Autowired
+    public void setTwcRevisionMmsCommitMapService(TwcRevisionMmsCommitMapService twcRevisionMmsCommitMapService) {
+        this.twcRevisionMmsCommitMapService = twcRevisionMmsCommitMapService;
+    }
+
+    @PutMapping(value = "/twc-revision/{revisionId}")
+    @PreAuthorize("@mss.hasProjectPrivilege(authentication, #projectId, 'PROJECT_EDIT', true)")
+    public CommitsResponse updateTwcRevisionID(
```

Comment:
does the key have to be twc-revisionId in the commit object or associated separately from element posts? there's currently 2 keys 'comment' and 'source' that are transferred over from element posts that can be used also, just fyi

Reply:
oh, i guess you don't know the twc revision id until after the elements are posted to mms...